### ISSUE TITLE
mask comparison in niak_brick_connectome_multiscale breaks

### ISSUE THREAD

**Issue by surchs** [SIMEXP/niak#296]
[niak_brick_connectome_multiscale](https://github.com/SIMEXP/niak/blob/master/extensions/glm_connectome/niak_brick_connectome_multiscale.m#L495) reads optional extra header information called `mask_suppressed` and `mask_scrubbing` and then performs a logical or on them. This breaks in Octave (4.0.2) and Matlab (2016b) if the two masks have different sizes (i.e. they have different length).

```
mx_el_or: nonconformant arguments (op1 is 122x1, op2 is 118x1)
File /usr/local/niak/extensions/glm_connectome/niak_brick_connectome_multiscale.m at line 495
File /usr/local/niak/extensions/glm_connectome/niak_brick_connectome_multiscale.m at line 377
File /usr/local/niak/extensions/psom-2.2.2/psom_run_job.m at line 185
File /usr/local/niak/extensions/psom-2.2.2/psom_run_job.m at line 106
File /usr/local/niak/extensions/psom-2.2.2/psom_worker.m at line 161
```

I don't know if these masks are of different size by design or if that itself is a bug but I encountered them in the simons_vip dataset. At least there should probably be some kind of shape check that lets this break gracefully if them having different sizes is unintended.

**Comment by pbellec** [comment]
It is a bug. The volumes of `mask_suppressed` are actually removed from the time series, and are not listed in `mask_scrubbing`. The only reason we did not catch that before is because we don't use `mask_suppressed` much....

**Comment by surchs** [comment]
so can we just comment this stuff out/delete it and go on with our lives?

**Comment by pbellec** [comment]
get rid of mask_suppressed yes. It is important to mask out volumes from mask_scrubbing though.

**Comment by poquirion** [comment]
fix in 0.18.0
